<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="automationframework" duration-ms="2330" started-at="2019-11-17T13:05:44Z" finished-at="2019-11-17T13:05:47Z">
    <groups>
    </groups>
    <test name="loginTest" duration-ms="2330" started-at="2019-11-17T13:05:44Z" finished-at="2019-11-17T13:05:47Z">
      <class name="com.automationFramework.testCases.TC_loginTest_001">
        <test-method status="PASS" signature="setup(java.lang.String)[pri:0, instance:com.automationFramework.testCases.TC_loginTest_001@146ba0ac]" name="setup" is-config="true" duration-ms="1789" started-at="2019-11-17T13:05:45Z" finished-at="2019-11-17T13:05:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="loginTest()[pri:0, instance:com.automationFramework.testCases.TC_loginTest_001@146ba0ac]" name="loginTest" duration-ms="17" started-at="2019-11-17T13:05:46Z" finished-at="2019-11-17T13:05:46Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.automationFramework.testCases.TC_loginTest_001.loginTest(TC_loginTest_001.java:14)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.automationFramework.testCases.TC_loginTest_001@146ba0ac]" name="tearDown" is-config="true" duration-ms="181" started-at="2019-11-17T13:05:46Z" finished-at="2019-11-17T13:05:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.automationFramework.testCases.TC_loginTest_001 -->
    </test> <!-- loginTest -->
  </suite> <!-- automationframework -->
</testng-results>
